app/addons/documents/doc-editor/components.react.jsx

```diff
@@ -227,7 +227,7 @@ var AttachmentsPanelButton = React.createClass({
     var doc = this.props.doc.get('_id');
 
     return _.map(this.props.doc.get('_attachments'), function (item, filename) {
-      var url = FauxtonAPI.urls('document', 'attachment', db, doc, app.utils.safeURLName(filename));
+      var url = FauxtonAPI.urls('document', 'attachment', encodeURIComponent(db), doc, app.utils.safeURLName(filename));
```

Comment:
Could you remove `safeURLName` we don't need it if we use the encodeURIComponent.